generated/nidcpower/nidcpower.proto

```diff
@@ -1,6 +1,6 @@
 
 //---------------------------------------------------------------------
-// This file is generated from NI-DCPower API metadata version 23.0.0d288
+// This file is generated from NI-DCPower API metadata version 23.0.0d318
```

Comment:
Note: These functions weren't present in the previous release of grpc-device and were marked to not codegen in this [AzDo PR](https://dev.azure.com/ni/DevCentral/_git/ni-central/pullrequest/392729?_a=files&path=/src/nidcpower/nidcpowerBaseDistribution/nidcpowerAPIMetadata/source/nidcpowerAPIMetadata/grpc_device/functions/functions.hapigen).

The resulting nimi-python metadata was updated based on those changes but the grpc-device repo was never updated so that's what these changes represent.

I believe these methods will be enabled before the 23.0 driver release so there should be another update to enable them here and in nimi-python. But until then, we should stay in sync with nimi-python and update to the same hapigen-generated metadata.

Reply:
Also, this matches the [proto file that's in nimi-python right now ](https://github.com/ni/nimi-python/blob/master/src/nidcpower/metadata/nidcpower.proto)(sourced on 23.0.0d318)

Reply:
I believe the DCPower methods will be exposed for gRPC only after 23.0 releases, in 23.3 for example.